R/PlatinumGenomes-QC/sql/check-sex.sql

```diff
@@ -1,35 +1,51 @@
-# Compute the the homozygous and heterozygous variant counts for each individual
-# within chromosome X to help determine whether the sex phenotype value is
-# correct for each individual.
+# Compute homozygosity (inbreeding) rate on chromosome X for each individual,
+# to help determine the correctness of the sex phenotype value.
 SELECT
   call.call_set_name,
-  ROUND(SUM(het_RA)/(SUM(hom_AA) + SUM(het_RA)), 3) AS perct_het_alt_in_snvs,
-  ROUND(SUM(hom_AA)/(SUM(hom_AA) + SUM(het_RA)), 3) AS perct_hom_alt_in_snvs,
-  SUM(hom_AA) AS hom_AA_count,
-  SUM(het_RA) AS het_RA_count,
-  SUM(hom_RR) AS hom_RR_count,
+  O_HOM,
+  ROUND(E_HOM, 2) as E_HOM,
+  N_SITES,
+  ROUND((O_HOM - E_HOM) / (N_SITES - E_HOM), 5) AS F
 FROM (
   SELECT
-    reference_bases,
-    GROUP_CONCAT(alternate_bases) WITHIN RECORD AS alternate_bases,
-    COUNT(alternate_bases) WITHIN RECORD AS num_alts,
     call.call_set_name,
-    SOME(call.genotype = 0) AND NOT SOME(call.genotype > 0) WITHIN call AS hom_RR,
-    SOME(call.genotype > 0) AND NOT SOME(call.genotype = 0) WITHIN call AS hom_AA,
-    SOME(call.genotype > 0) AND SOME(call.genotype = 0) WITHIN call AS het_RA
-  FROM
-    [_MULTISAMPLE_VARIANT_TABLE_]
-  WHERE
-    (reference_name = 'chrX' OR reference_name = 'X')
-    AND start NOT BETWEEN 59999 AND 2699519
-    AND start NOT BETWEEN 154931042 AND 155260559
-  HAVING
-    # Skip 1/2 genotypes _and non-SNP variants
-    num_alts = 1
-    AND reference_bases IN ('A','C','G','T')
-    AND alternate_bases IN ('A','C','G','T')
-  )
-GROUP BY
-  call.call_set_name
+    SUM(first_allele = second_allele) AS O_HOM,
+    SUM(1.0 - (2.0 * freq * (1.0 - freq) * (called_allele_count / (called_allele_count - 1.0)))) AS E_HOM,
```

Comment:
Is this query exactly the same as https://github.com/googlegenomics/codelabs/blob/master/R/PlatinumGenomes-QC/sql/homozygous-variants.sql except for the WHERE clause?